internal/resources/kustomization/resource_kustomization_mock_test.go

```diff
@@ -0,0 +1,432 @@
+			ManagementClusterName: "attached",
+		},
+		Spec: &kustomizationSpec,
+		Meta: &objectmetamodel.VmwareTanzuCoreV1alpha1ObjectMeta{
+			ParentReferences: referenceArray,
+			Description:      "resource with description",
+			Labels: map[string]string{
+				"key1": "value1",
+				"key2": "value2",
+			},
+			UID:             "kustomization1",
+			ResourceVersion: "v1",
+		},
+		Status: &kustomizationclustermodel.VmwareTanzuManageV1alpha1ClusterNamespaceFluxcdKustomizationStatus{
+			Conditions: map[string]statusmodel.VmwareTanzuCoreV1alpha1StatusCondition{
+				"Ready": {
+					Reason: "made successfully",
+				},
+			},
+		},
+	}
+
+	getResponse := &kustomizationclustermodel.VmwareTanzuManageV1alpha1ClusterNamespaceFluxcdKustomizationGetKustomizationResponse{
+		Kustomization: getModel,
+	}
+
+	postRequestContinuousDeliveryModel := &continuousdeliveryclustermodel.VmwareTanzuManageV1alpha1ClusterFluxcdContinuousdeliveryContinuousDelivery{
+		FullName: &continuousdeliveryclustermodel.VmwareTanzuManageV1alpha1ClusterFluxcdContinuousdeliveryFullName{
+			ClusterName: testConfig.ScopeHelperResources.Cluster.Name,
+			OrgID:       OrgID,
+		},
+		Meta: &objectmetamodel.VmwareTanzuCoreV1alpha1ObjectMeta{
+			ParentReferences: nil,
+			Description:      "resource with description",
+			Labels: map[string]string{
+				"key1": "value1",
+				"key2": "value2",
+			},
+			UID:             "continuousdelivery1",
+			ResourceVersion: "v1",
+		},
+	}
+
+	postResponseContinuousDelivery := &continuousdeliveryclustermodel.VmwareTanzuManageV1alpha1ClusterFluxcdContinuousdeliveryContinuousDelivery{
+		FullName: &continuousdeliveryclustermodel.VmwareTanzuManageV1alpha1ClusterFluxcdContinuousdeliveryFullName{
+			ClusterName: testConfig.ScopeHelperResources.Cluster.Name,
+			OrgID:       OrgID,
+		},
+		Meta: &objectmetamodel.VmwareTanzuCoreV1alpha1ObjectMeta{
+			ParentReferences: nil,
+			Description:      "resource with description",
+			Labels: map[string]string{
+				"key1": "value1",
+				"key2": "value2",
+			},
+			UID:             "continuousdelivery1",
+			ResourceVersion: "v1",
+		},
+		Status: &continuousdeliveryclustermodel.VmwareTanzuManageV1alpha1ClusterFluxcdContinuousdeliveryStatus{
+			Conditions: map[string]statusmodel.VmwareTanzuCoreV1alpha1StatusCondition{
+				"Ready": {
+					Reason: "made successfully",
+				},
+			},
+		},
+	}
+
+	postContinuousDeliveryRequest := &continuousdeliveryclustermodel.VmwareTanzuManageV1alpha1ClusterFluxcdContinuousdeliveryContinuousDeliveryRequest{
+		ContinuousDelivery: postRequestContinuousDeliveryModel,
+	}
+
+	postContinuousDeliveryResponse := &continuousdeliveryclustermodel.VmwareTanzuManageV1alpha1ClusterFluxcdContinuousdeliveryContinuousDeliveryResponse{
+		ContinuousDelivery: postResponseContinuousDelivery,
+	}
+
+	postEndpoint := fmt.Sprintf("https://%s/v1alpha1/clusters/%s/namespaces/%s/fluxcd/kustomizations", endpoint, testConfig.ScopeHelperResources.Cluster.Name, testConfig.Namespace)
```

Comment:
Like previous MR, use the existing helper to construct URL

Reply:
updated PTAL

Reply:
updated PTAL